packages/page-democracy/src/Overview/Referendums.tsx

```diff
@@ -25,6 +25,7 @@ function Referendums ({ className = '', referendums }: Props): React.ReactElemen
     [t('turnout'), 'media--1400'],
     [undefined, 'badge'],
     [t('votes'), 'expand'],
+    [t('aye percentage'), 'media--1000'],
```

Comment:
```suggestion
    [undefined, 'media--1000'],
```

The reason here - the label is much wider than the actual data in the column, so it just adds extra whitespace. Additionally it doesn't actually add any information that the graph doesn't convey.